conda/cli/activate.py

```diff
@@ -62,7 +81,7 @@ def main():
             print(os.environ['PATH'])
             raise
         paths = []
-        sys.stderr.write("discarding %s from PATH\n" % binpath)
+        sys.stderr.write("discarding %s from PATH\n" % pathlist_to_str(binpath))
```

Comment:
In contrast to the old design, this command also removes the root environment's `Scripts` directory. Since `activate.bat` and `deactivate.bat` only exist in the root environment (because `conda` can only reside there), these commands are no longer available after the first `activate some_env` call. The error was initially masked because `conda ..deactivate` never succeeded in the root environment due to the missing `Library\bin` subdirectory. 

The upshot is that the root environment's `Scripts` directory must always remain in the PATH, but it must come after the `Scripts` directory of the active environment.